ocs_ci/ocs/ui/views.py

```
@@ -656,11 +656,16 @@
         "//label[@for='region']/../following-sibling::*",
         By.XPATH,
     ),
+    "store_switch_to_secret": (
+        "//*[normalize-space()='Switch to Secret']",
+        By.XPATH,
+    ),
     "store_secret_dropdown": (
-        "//span[@class='text-muted' and text()='Select Secret']/../..",
+        "//label[normalize-space()='Secret Key']/../following-sibling::*//button[contains(@class, 'c-menu-toggle')] | "
+        "//span[normalize-space()='Select Secret']/ancestor::button[contains(@class, 'c-menu-toggle')]",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)